lib/Parse/ParseIfConfig.cpp

```diff
@@ -53,6 +53,17 @@ static StringRef extractExprSource(SourceManager &SM, Expr *E) {
   return SM.extractText(Range);
 }
 
+static bool isValidPrefixUnaryOperator(llvm::Optional<StringRef> &UnaryOperator) {
+  return UnaryOperator != None && (UnaryOperator.getValue() == ">=" || UnaryOperator.getValue() == "<");
+}
+
+static bool isValidVersion(version::Version &Version, version::Version &ExpectedVersion, StringRef UnaryOperator) {
```

Comment:
Nitpick: These should be `const` references, and please wrap at 80 columns. (`clang-format` should be set up correctly for this repo.)

Reply:
Ah, thanks. I've run `git clang-format` to not pollute the diff.